features/fixtures/handled/browserify/a.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="dist/a.js"></script>
```

Comment:
What do a/b/c stand for in the JS + HTML files - is there a more descriptive naming scheme that would make sense to use?